packages/lib-panoptes-js/config.js

```diff
@@ -0,0 +1,55 @@
+/*
+Configuration Settings
```

Comment:
This file contains code, so it should be tested as well, even if it's only to check that the expected config object and env string is present.